tests/aci-e2e/e2e-aci_test.go

```diff
@@ -456,22 +469,36 @@ func TestContainerRunAttached(t *testing.T) {
 
 func TestComposeUpUpdate(t *testing.T) {
 	c := NewParallelE2eCLI(t, binDir)
-	_, groupID := setupTestResourceGroup(t, c)
+	sID, groupID := setupTestResourceGroup(t, c)
 
 	const (
-		composeFile              = "../composefiles/aci-demo/aci_demo_port.yaml"
-		composeFileMultiplePorts = "../composefiles/aci-demo/aci_demo_multi_port.yaml"
-		composeProjectName       = "acidemo"
-		serverContainer          = composeProjectName + "_web"
-		wordsContainer           = composeProjectName + "_words"
-		dbContainer              = composeProjectName + "_db"
+		composeFileVolumes              = "../composefiles/aci-demo/aci_demo_port_volumes.yaml"
+		composeFileVolumesMultiplePorts = "../composefiles/aci-demo/aci_demo_multiport_volumes.yaml"
+		composeProjectName              = "acidemo"
+		serverContainer                 = composeProjectName + "_web"
+		wordsContainer                  = composeProjectName + "_words"
+		dbContainer                     = composeProjectName + "_db"
+		composeAccountName              = "dockertestvolumeaccount"
```

Comment:
We need to derive a storage account name from the resource group name, t be able to run thing in //. This will require modifying the content of the compose file